.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       with:
         submodules: true
     - name: Install Rust
-      run: rustup update stable && rustup default stable
+      run: rustup update --no-self-update stable && rustup default stable
```

Comment:
This is a workaround for a Github Actions windows-2022 image bug; see https://github.com/actions/runner-images/issues/7491